src/libPMacc/include/pmacc_types.hpp

```diff
@@ -176,16 +178,14 @@ enum EventType
  * you must align all array and structs which can used on device
  * @param byte byte of data which must aligned
```

Comment:
actually we should check if the `must align` is now actually a `should align for performance`
( @psychocoderHPC  remembers that in sm_13 non-aligned data did cause bugs)